clients/cloud/java-springboot/scripts/ccloud-generate-cp-configs.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
This file is already in this repo at https://github.com/confluentinc/examples/blob/5.4.1-post/ccloud/ccloud-generate-cp-configs.sh, can you just use the existing one?

Reply:
done. switched to use standard scripts